usr.sbin/inetd/inetd.h

```diff
@@ -122,7 +122,7 @@ struct	servtab {
 	char	*se_hostaddr;		/* host address to listen on */
 	char	*se_service;		/* name of service */
 	int	se_socktype;		/* type of socket to use */
-	int	se_family;		/* address family */
+	sa_family_t	se_family;	/* address family */
```

Comment:
sa_family_t might be smaller (an unsigned short) than the previous int value but I don't think anything relies on the exact struct layout and we already changed se_count and other fields.